docs/configuring-playbook-matrix-media-repo.md

```diff
@@ -18,6 +18,7 @@ Add the following configuration to your `inventory/host_vars/matrix.DOMAIN/vars.
 
 ```yaml
 matrix_media_repo_enabled: true
+matrix_media_repo_database_password: "your_password"
```

Comment:
Why explicitly ask for a database password here, when passwords for all components are auto-generated via `group_vars/matrix_servers`?

https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/64db27c7faf3032341c665f8abf7aa66c7aa7aca/group_vars/matrix_servers#L2706

Reply:
Ah good catch. Forgot about that while looking over the default values. Removed the placeholder value from the docs and defaults.